mockgen/parse.go

```diff
@@ -292,7 +292,9 @@ func (p *fileParser) parseInterface(name, pkg string, it *ast.InterfaceType) (*m
 			}
 			// Copy the methods.
 			// TODO: apply shadowing rules.
```

Comment:
Not sure what "shadowing rules" are. Does this refer to overlapping methods? Should I remove this TODO?

Reply:
I will need to look into this and get back to you.